journal/updated/02-chapter-1.adoc

```diff
@@ -201,65 +204,66 @@ supposing you might be at the Quarterly Meeting.
 I hope you will not be negligent in attending such meetings,
 nor any other which are appointed for the service of Truth,
 and for the worship of Almighty God.
-I pray you, let not the indifferency of others lull you asleep;
+I beseech you, let not the indifferency of others lull you asleep,
 but be diligent and watchful, truly waiting in the depth of humility,
-and then I doubt not but you will be favoured with the presence of God,
+and then I doubt not that you will be favoured with the presence of God,
 both to your comfort and edification;
-for it is the Divine Monitor from heaven that is still the teacher of his people.
-It is He that yet speaks as never man spoke, and he cannot be removed into a corner.
-I hope dear cousin, you are sensible of these things,
+for it is the Divine Monitor from heaven that is still the teacher of His people.
+It is He that yet speaks as never man spoke, and He cannot be removed into a corner.
```

Comment:
lets put this in quotes (because very few know this verse). Like this: and "He cannot be removed into a corner." and then put the reference in a footnote, which is Isaiah 30:20.